### ISSUE TITLE
spelling of potato

### ISSUE THREAD

**Issue by Erreinion** [HabitRPG/habitica#2431]
In the confirmation window when feeding a potato to a pet, it is spelled 'potatoe'

**Comment by wc8** [comment]
There’s a user request to leave it spelled wrong… I think a proposed quest might be factoring that mistake in. 
I tend to think we should correct it when we get a chance, for the sake of users who are learning English, and the quest can factor in the correction.

**Comment by hattieshoe** [comment]
Even though I'm one of those who's enjoyed the misspelling and joked about it, I agree with wc8 that it would be better to correct it and not leave it just for the sake of nostalgia for a few people. Most readers would just assume it was a typo which isn't the impression you want to give. I'm sure the proposed quest could easily work around the correction.

**Comment by katerberg** [comment]
When I grep the repository, I don't see this spelling anywhere in the code. I was just looking for some low-hanging fruit (or potatoes...) to clean up.

I assume it's in the database rather than in the actual code base?

**Comment by colegleason** [comment]
It's [here](https://github.com/HabitRPG/habitrpg-shared/blob/c9ddf0e15dbf8bf8a2b8137d0a8e8a020efc2cb9/script/content.coffee#L510). Maybe we should also fix the modal to use the text attribute.

**Comment by katerberg** [comment]
I have the fix done but I don't know how to submit a pull request. I have it completed locally on my own branch but I get 403s when trying to push my branch. Do I need to fork the repo? This is my first time working on a group project on github. Sorry :baby:

**Comment by colegleason** [comment]
Yeah, here is what you should do.
- Fork the repo on Github
- Locally where you have the changes, run:

```
git remote rename origin upstream
git remote add origin git@github.com:diablomarcus/habitrpg-shared.git
git push origin develop:potato
```
- Go to habitrpg/habitrpg-shared and open a PR.

**Comment by katerberg** [comment]
Submitted! https://github.com/HabitRPG/habitrpg/pull/2439

I did not actually change the key name since I don't know if it's stored in the DB or somewhere else that persists and the text should be the only user-visible part.

**Comment by deilann** [comment]
If we're going to fix this (and give me profound sadness, of the cats wearing clothes and flavored coffee beans types) the backwards nature of the feeding message should probably also be fixed. (Talks about Wolf-Base or Wolf-Desert instead of Base Wolf or Desert Wolf.)

**Comment by katerberg** [comment]
@deilann This has been included on the pull request as well.

**Comment by colegleason** [comment]
Fixed via #2439
